doc/users/next_whats_new/mouse_rotation.rst

```diff
@@ -4,9 +4,12 @@ Rotating 3d plots with the mouse
 Rotating three-dimensional plots with the mouse has been made more intuitive.
 The plot now reacts the same way to mouse movement, independent of the
 particular orientation at hand; and it is possible to control all 3 rotational
-degrees of freedom (azimuth, elevation, and roll). It uses a variation on
-Ken Shoemake's ARCBALL [Shoemake1992]_.
+degrees of freedom (azimuth, elevation, and roll). By default,
+it uses a variation on Ken Shoemake's ARCBALL [1]_.
```

Comment:
Should include a code snippet with the options, and showing how to revert back to the previous behavior.

Reply:
It makes sense to show here how to revert back to the original style, it's a good suggestion, I'll do that; but I feel less certain about including a 'code snippet with the options' (I'd be just duplicating view_angles#rotation-with-mouse, it feels like). 
How would you include that?

Reply:
I think duplicating information in the "What's New" section is fine - in general I see no issue with overexplaining but in this case including things in the "changelog" is especially helpful since it makes it more immediately accessible to users trying to figure out why things changed.

Reply:
I'll add the code snippet from the "Rotation with the mouse" part; take a look if you find it sufficient.

Reply:
Looks good to me!